### PR TITLE
bradl3yC - 9867 - Fix dl nesting on review page

### PR DESCRIPTION
**Pull request by bradl3yC**
## Description
Axe scan revealed an error with proper element nesting dl / dt / dd and friends

## Screenshots
passing axe (with relation to dd nesting - just pretend the heading level stuff isn't there - its not related):
Screen Shot 2020-06-18 at 2 48 22 PM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/2346/sass/form-2346.scss [hunk 1]
```diff
@@ -69,3 +69,23 @@ dt {
...
+  border-bottom: 1px solid $color-gray-light;
+  .review-row {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    border-top: 1px solid $color-gray-light;
+    padding: 1.5rem;
+    padding-left: 0;
+  }
+  .review-row dd {
```
Comment (kelsonic) [inline src/applications/disability-benefits/2346/sass/form-2346.scss:83]: You could nest these within the `.review-row {` selector on line 75 if you want 🙂

File: src/applications/disability-benefits/2346/schemas/2346UI.js [hunk 2]
```diff
@@ -105,6 +105,7 @@ export default {
       'ui:options': {
         classNames: 'vads-u-display--none',
         hideOnReview: true,
+        customTitle: ' ',
```
Comment (kelsonic) [inline src/applications/disability-benefits/2346/schemas/2346UI.js:108]: Just confirming you meant to put a space in that empty string here 🙂

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉

File: src/applications/disability-benefits/2346/schemas/2346UI.js (hunk 2 above)
Comment (bradl3yC) [reply src/applications/disability-benefits/2346/schemas/2346UI.js:108]: can confirm - necessary - yolo forms system

**Review by maharielrosario — APPROVED** [review]

**Review by LinnJS — APPROVED** [review]

**Pull request merged**